modules/dedicated-managing-dedicated-administrators.adoc

```diff
@@ -6,24 +6,11 @@
 =  Managing {product-title} administrators
 
 Administrator roles are managed using a `dedicated-admins` group on the cluster.
-Existing members of this group can edit membership. To view a list of current
-dedicated administrators by user name, you can use the following command:
+Existing members of this group can edit membership via the OpenShift Cluster
+Manager.
 
-----
-$ oc describe group dedicated-admins
-----
-
-To add a new member to the `dedicated-admins` group:
-
-----
-$ oc adm groups add-users dedicated-admins <user_name>
-----
-
-To remove an existing user from the `dedicated-admins` group:
-
-----
-$ oc adm groups remove-users dedicated-admins <user_name>
-----
+. In *Cluster Details*, click *Users* -> *Add User*.
+. Enter the *Username* and *Group* (dedicated-admins), then click the *Add* button.
```

Comment:
- only click "Add User" for the first user
- missing how to remove a user

Reply:
I removed the step *Add User*. Can you provide me steps to remove a user?

Reply:
In the "Users" tab click the X to the right of the user + group you'd like to remove.

![Screenshot from 2019-08-13 18-34-30](https://user-images.githubusercontent.com/480388/62982043-0468e380-bdf9-11e9-8511-e25dc100206b.png)

Reply:
When adding the first user you have to click the "Add User" button.  Here's a screenshot.

![Screenshot from 2019-08-20 16-21-43](https://user-images.githubusercontent.com/480388/63381435-ad1dc280-c366-11e9-8de0-a1f7c7b88170.png)

Reply:
Updated the steps again, thanks.